frontend/src/app/patients/ManagePatients.scss

```diff
@@ -27,3 +27,7 @@
 .patient-form-deletion-field {
   width: calc(100% + 2.5rem);
 }
+
+.cursor-pointer {
```

Comment:
I don't think you need this since `cursor-pointer` provided through uswds https://designsystem.digital.gov/utilities/